docs/src/quickstart/cythonize.rst

```diff
@@ -55,27 +69,40 @@ Typing Functions
 
 Python function calls can be expensive -- in Cython doubly so because
 one might need to convert to and from Python objects to do the call.
-In our example above, the argument is assumed to be a C double both inside f()
+In our example above, the argument is assumed to be a C double both inside ``f()``
 and in the call to it, yet a Python ``float`` object must be constructed around the
 argument in order to pass it.
 
-Therefore Cython provides a syntax for declaring a C-style function,
-the cdef keyword:
+Therefore Cython language provides a syntax for declaring a C-style function,
+the ``cdef`` keyword. Moreover, Cython library provides ``@cfunc`` decorator to
+declare C-style function without need of using Cython language. Both aproaches are
+equivalent and produces same C code:
+
+.. tabs::
+
+    .. group-tab:: Pure Python
 
-.. literalinclude:: ../../examples/quickstart/cythonize/cdef_keyword.pyx
+        .. literalinclude:: ../../examples/quickstart/cythonize/cdef_keyword.py
+
+    .. group-tab:: Cython
+
+        .. literalinclude:: ../../examples/quickstart/cythonize/cdef_keyword.pyx
 
 Some form of except-modifier should usually be added, otherwise Cython
 will not be able to propagate exceptions raised in the function (or a
 function it calls). The ``except? -2`` means that an error will be checked
 for if ``-2`` is returned (though the ``?`` indicates that ``-2`` may also
-be used as a valid return value).
+be used as a valid return value). The same can be expressed using only python
+language by ``@exceptval(-2, check=True)`` decorator.
```

Comment:
Please mind the spelling of "Python" (and Cython) as a name with an upper case initial.
```suggestion
be used as a valid return value). The same can be expressed using only Python
syntax with the decorator ``@exceptval(-2, check=True)``.
```